samples/apps/copilot-chat-app/scripts/Install-Requirements.ps1

```diff
@@ -0,0 +1,21 @@
+<#
+.SYNOPSIS
+Install the requirements for running Copilot Chat. Note that this script only works on Windows.
+#>
+
+if ($IsWindows)
+{
+    # Install chocolatey
+    Set-ExecutionPolicy Bypass
+    [System.Net.ServicePointManager]::SecurityProtocol = 3072
+    Invoke-Expression (New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1')
+
+    $env:PATH += ";%ALLUSERSPROFILE%\chocolatey\bin"
+
+    # Install packages
+    choco install -y dotnet-6.0-sdk nodejs yarn
```

Comment:
This seems to work fine when I have everything already installed, but could explicitly check and skip whatever is already there.